### ISSUE TITLE
GUI doesn't seem to work (irresponsive) on Manjaro XFCE

### ISSUE THREAD

**Issue by nhmacuk** [snyball/Hawck#17]
When issuing the hawck-ui command following the README-instructions, the GUI comes up. However it seems unusable; the different sections are clickable, but it seems unable to change values within the sections. I get the following response in the terminal:

```
(hawck-ui:2205): Gtk-CRITICAL **: 14:20:24.901: gtk_widget_grab_default: assertion 'gtk_widget_get_can_default (widget)' failed
1898
1901
<hawck_ui.window.Settings object at 0x7f7a20a101c0> autostart_switch False
<hawck_ui.window.Settings object at 0x7f7a20a101c0> unsafe_mode_switch False
handler_name: on_notify_on_err_chk_toggled
Unable to retrieve config.stop_on_err
handler_name: on_eval_keydown_chk_toggled
handler_name: on_eval_keyup_chk_toggled
Unable to retrieve config.eval_keyrepeat
Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/gi/overrides/Gtk.py", line 110, in _builder_connect_callback
    handler, args = _extract_handler_and_args(obj_or_map, handler_name)
  File "/usr/lib/python3.8/site-packages/gi/overrides/Gtk.py", line 90, in _extract_handler_and_args
    raise AttributeError('Handler %s not found' % handler_name)
AttributeError: Handler on_stop_on_err_chk_toggled not found
Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/gi/overrides/Gtk.py", line 110, in _builder_connect_callback
    handler, args = _extract_handler_and_args(obj_or_map, handler_name)
  File "/usr/lib/python3.8/site-packages/gi/overrides/Gtk.py", line 90, in _extract_handler_and_args
    raise AttributeError('Handler %s not found' % handler_name)
AttributeError: Handler on_eval_keyrepeat_chk_toggled not found
```
![image](https://user-images.githubusercontent.com/44981227/71518923-7c6e0c80-28b5-11ea-9201-7e2fa738cc96.png)

**Comment by snyball** [comment]
Sorry for the late response, been too busy to check the issues here.

Nothing in the terminal output indicates a serious problem, the errors are related to a mismatch between the UI XML and the Python code, but this mismatch isn't serious enough that it actually causes any problems (I get the same output as you.)

So unfortunately I don't really know where to go from here to debug this, the output does show "Focus change" which means that the Python callbacks are actually running and changing the state of the UI. The UI not rendering its changes to the screen could be caused by a newer version of GTK+ that my code is incompatible with, or something else entirely.

Basically I have no idea what causes this with the available information.

Hope you were able to resolve it somehow, if it doesn't resolve itself, then you can also use Hawck without the UI: https://github.com/snyball/Hawck/issues/13#issuecomment-524439345

**Comment by snyball** [comment]
Closing this issue as I'm planning to deprecate the UI by version 1.0
